samples/apps/forum/src/controllers/site.ts

```diff
@@ -41,20 +41,27 @@ const HEADER_HTML = `
   <script src="//code.jquery.com/jquery-3.5.1.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
   <script src="//cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/js/bootstrap.bundle.min.js" integrity="sha384-ho+j7jyWK8fNQe+A12Hb8AhRq26LrZ/JpcUGGOn+Y7RsweNrtN/tE3MoK7ZeZDyx" crossorigin="anonymous"></script>
   <script src="//cdn.jsdelivr.net/npm/js-cookie@3.0.0-rc.1/dist/js.cookie.min.js"></script>
-  <script src="//cdnjs.cloudflare.com/ajax/libs/crypto-js/4.0.0/crypto-js.min.js"></script>
   <script src="//cdn.jsdelivr.net/npm/jstat@1.9.4/dist/jstat.min.js"></script>
   <script src="//cdn.plot.ly/plotly-1.57.0.min.js"></script>
   <script src="//unpkg.com/papaparse@5.3.0/papaparse.min.js"></script>
-  <script src="//alcdn.msauth.net/browser/2.1.0/js/msal-browser.min.js"></script>
+  <script src="//alcdn.msauth.net/browser/2.4.1/js/msal-browser.min.js"></script>
   <script src="//cdn.jsdelivr.net/npm/jwt-decode@3.0.0/build/jwt-decode.js"></script>
   <script>
 const apiUrl = window.location.origin + '/app/polls'
 const siteUrl = window.location.origin + '/app/site'
 
+// "CCF Demo App" app registration
+const MS_APP_ID = "1773214f-72b8-48f9-ae18-81e30fab04db"
+// Note: Even though a scope is not strictly required for this app,
+// it is needed so that Microsoft produces standard-compliant JWTs.
+// Creating and requesting a scope of our own app changes the token audience
+// to our app instead of being Microsoft (which is the default).
+// See also https://xsreality.medium.com/making-azure-ad-oidc-compliant-5734b70c43ff.
+const MS_LOGIN_SCOPES = ["api://1773214f-72b8-48f9-ae18-81e30fab04db/Polls.Access"]
```

Comment:
Perhaps add a link to https://docs.microsoft.com/en-us/graph/auth-register-app-v2, which is a pre-requisite to get any similar sample app working?

Reply:
This is for Graph API specifically which may have confusing bits. I added a link to https://docs.microsoft.com/en-us/azure/active-directory/develop/scenario-protected-web-api-overview instead which is likely a better landing page.